cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,120 @@
+        instance_detail = "Instance ID: " + instance['InstanceId']
+        instance_detail += ", Type: " + instance.get('InstanceType', 'Unknown')
+        instance_detail += ", State: " + instance.get('State', {}).get('Name', 'Unknown')
+        instance_details.append(instance_detail)
+
+if instance_details:
+    result["message"] = "Found " + str(len(instance_details)) + " EC2 instances:\n" + "\n".join(instance_details)
+else:
+    result["message"] = "No EC2 instances found"
+```
+
+6. **Save the Command**
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/1/1

(MD029, ol-prefix)